### PR TITLE
🔧 collection mobile

### LINKED ISSUE #10572: Collection mobile followup
ref: 
- https://github.com/kodadot/nft-gallery/issues/10564

### PR DESCRIPTION
**Pull request by roiLeo**
## PR Type

- [x] Refactoring
- [x] css to tailwind

## Needs Design check

- @exezbcz please review

## Needs QA check

- @kodadot/qa-guild please review

## Context

- [x] Same as #10564 
- [x] Closes #10572 

## Screenshot 📸

Capture d’écran 2024-07-03 à 12 52 17 PM

### CODE REVIEW

**Review by Jarsen136 — APPROVED** [review]

**Review by hassnian — COMMENTED** [review]
follow up of #10564 , profile banner skeleton needs to be updated   

can you also fix this here?. 

thanks

CleanShot 2024-07-04 at 14 58 19

**Comment by roiLeo** [comment]
Capture d’écran 2024-07-04 à 2 00 06 PM

File: components/collection/CollectionHeader/CollectionBanner.vue [hunk 1]
```diff
@@ -1,22 +1,23 @@
...
+            <div class="collection-banner-avatar p-2.5 mb-4 md:mb-6">
               <BaseMediaItem
                 :src="collectionAvatar"
                 :image-component="NuxtImg"
                 :title="collectionName"
                 class="w-[5.5rem] h-[5.5rem] border" />
             </div>
             <h1
-              class="collection-banner-name"
+              class="collection-banner-name font-bold title is-4 md:is-3"
```
Comment (hassnian) [inline components/collection/CollectionHeader/CollectionBanner.vue:20]: no bulma classes, can you use tailwind instate

**Review by hassnian — CHANGES_REQUESTED** [review]
something broke

**Review by hassnian — CHANGES_REQUESTED** [review]
something broke

**Comment by hassnian** [comment]
CleanShot 2024-07-06 at 09 35 09@2x

**Comment by roiLeo** [comment]
> profile picture scale down, from 144px to 100px, centered vertically in the banner

where the 40px padding comes from?

**Comment by hassnian** [comment]
checked canary and it has the same issues, would appreciate if you could fix them , thanks

## profile

### desktop 

1. image banner padding should be (40px)
2. profile title should be 31px

https://github.com/kodadot/nft-gallery/assets/44554284/126dd4f6-f806-4686-aa9b-24d8e1a05fd3

### mobile

looks good

**Comment by roiLeo** [comment]
> ## profile
> ### desktop
> 
>     1. image banner padding should be (40px)
>     2. profile title should be 31px

thank you for these clarifications, should be good now.
IMO it would be better to customize tailwind fontSize class than have specific size.

**Comment by hassnian** [comment]
> thank you for these clarifications, should be good now. 

looking good, thanks! 

> IMO it would be better to customize tailwind fontSize class than have specific size.

if you want #10611  :)

**Review by hassnian — APPROVED** [review]

**Pull request merged**